templates/@theme-base/_styles.scss

```diff
@@ -8,6 +8,7 @@ $bgcolor-searchbox-field: $search;
 $bgcolor-spacebar: rgba(255, 255, 255, 0.1);
 $bgcolor-aside-link: rgba(0, 0, 0, 0.3);
 $bgcolor-toolbar: lighten($base, 5%);
+$bgcolor-subnav: lighten($base, 5%);
```

Comment:
It seems protonmail is now using the variable --bgcolor-subnav itself:

<img width="227" alt="image" src="https://user-images.githubusercontent.com/5780704/70426591-97e2b600-1a73-11ea-9568-48a4eedcf4ac.png">

Does this mean they'll add it to their design system? For now I'll remove it, because it's not yet in their docs.

Reply:
I see, we'll skip implementing it for now as i[t is not on their master branch for the design system](https://github.com/ProtonMail/design-system/blob/master/_sass/pm-styles/_pm-theme-config.scss) just yet. Thanks for removing it.

I would like to audit the theme templates when PM `v4.0.0`is released to make more use of the appropriate CSS variables provided too.